cmd/docs/docs.go

```diff
@@ -0,0 +1,62 @@
+package docs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+type Options struct {
+	// the path to generate the documentation in
+	path string
+}
+
+const DefaultPath = "./docs"
+
+func NewDocsCommand() *cobra.Command {
+	opts := &Options{}
+
+	return &cobra.Command{
+		Use:   "docs",
+		Short: "Generates the documentation for the CLI",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
I'm not sure if this is best practices for Cobra.

Reply:
Yup, that's what those utility functions are for!! In a few places, I've defined our own function for `Args` that returns a `func(cmd *Command, args []string) error` and gives us more flexability:

https://github.com/open-sauced/pizza-cli/blob/1f6258c587979c4af5547e5123593417d1289705/cmd/generate/generate.go#L20

But either works great!